t/plugin/proxy-rewrite.t

```diff
@@ -1311,3 +1311,56 @@ host: test.com:6443
 x-real-ip: 127.0.0.1
 --- no_error_log
 [error]
+
+
+
+=== TEST 45: set route(rewrite host)
+--- config
```

Comment:
Let's add new test to https://github.com/apache/apisix/blob/master/t/plugin/proxy-rewrite3.t

Reply:
Done